sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -37,6 +44,50 @@ export function getRetryAttemptTimeoutInMs(retryOptions: RetryOptions | undefine
   return timeoutInMs;
 }
 
+/**
+ * The set of options to configure request cancellation.
+ * - `abortSignal` : A signal used to cancel an asynchronous operation.
+ */
+export interface AbortSignalOptions {
+  /**
+   * An implementation of the `AbortSignalLike` interface to signal the request to cancel the operation.
+   * For example, use the &commat;azure/abort-controller to create an `AbortSignal`.
+   */
+  abortSignal?: AbortSignalLike;
+}
+
+/**
+ * The set of options to manually propagate `Span` context for distributed tracing.
+ * - `parentSpan` : The `Span` or `SpanContext` for the operation to use as a `parent` when creating its own span.
+ */
+export interface ParentSpanOptions {
```

Comment:
this also looks familiar... maybe there are some common interfaces that need to be pulled out?

Reply:
Yeah, maybe this can be pulled into core-tracing. I think core-http has something similar on RequestOptions, but this library doesn't depend on that so I duplicated it here.